cocos/web-view/web-view-impl-web.ts

```diff
@@ -25,21 +25,17 @@
 import { screenAdapter } from 'pal/screen-adapter';
 import { error, warn } from '@base/debug';
 import { ccwindow } from '@base/global';
+import { isDescendantElementOf } from '@pal/utils';
 import { EventType } from './web-view-enums';
 import { WebViewImpl } from './web-view-impl';
 import { game } from '../game';
 import { mat4 } from '../core/math';
-import { contains } from '../core/utils/misc';
 
 const ccdocument = ccwindow.document;
 
 const _mat4_temp = mat4();
 
 export class WebViewImplWeb extends WebViewImpl {
-    constructor (component: any) {
-        super(component);
-    }
-
```

Comment:
Why remove the constructor?

Reply:
because this constructor is meanningless, but it introduce an `any` type of parameter

Reply:
Ok, please double check it is useless.